source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```
@@ -1868,14 +2046,16 @@ ul.subCategory {
 }
 .navigationSection {
   position: fixed;
+  width: 6rem;
+  font-size: 2rem;
+  background-color: var(--ca-legacy-background-color);
+  top: 19rem;
+  right: 0;
   bottom: 0;
   display: block;
```

Stylelint (17.7.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)